test/jpypetest/test_array.py

```diff
@@ -593,6 +593,7 @@ def testShortcut(self):
         # Check Objects
         self.assertEqual(JString[5].getClass(), JArray(JString)(5).getClass())
         self.assertEqual(JObject[5].getClass(), JArray(JObject)(5).getClass())
+        self.assertEqual(JClass[5].getClass(), JArray(JClass)(5).getClass())
```

Comment:
This seems reasonable.  Though it is a rather limited use case.  So long as it is consistent with the rest of the library I have no objections.